test/sequential/test-module-loading.js

```diff
@@ -69,6 +69,9 @@ assert.strictEqual(threeFolder, threeIndex);
 assert.notStrictEqual(threeFolder, three);
 
 console.error('test package.json require() loading');
+assert.throws(function() {
+  require('../fixtures/packages/invalid');
+}, /package.json: Unexpected token/);
```

Comment:
We treat changes to error messages as breaking changes. So, it's a good way to detect if anything changes.